consul_exporter.go

```diff
@@ -250,7 +263,11 @@ func main() {
 	log.Infoln("Starting consul_exporter", version.Info())
 	log.Infoln("Build context", version.BuildContext())
 
-	exporter := NewExporter(*consulServer, *kvPrefix, *kvFilter, *healthSummary)
+	exporter, err := NewExporter(*consulServer, *kvPrefix, *kvFilter, *healthSummary)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(-1)
```

Comment:
Our logging library supports [Fatalln](https://godoc.org/github.com/prometheus/common/log#Fatalln). Let's use this instead.